storage/netstore.go

```diff
@@ -201,7 +205,8 @@ func (n *NetStore) Get(ctx context.Context, mode chunk.ModeGet, req *Request) (c
 			if ok {
 				ch, err = n.RemoteFetch(ctx, req, fi)
 				if err != nil {
-					if n.recoveryCallback != nil {
+					if n.recoveryCallback != nil && publisher != "" {
+						log.Debug("gp netstore recovery triggered")
```

Comment:
same here

Reply:
This log could be useful for debugging purposes, do you mean to remove it? or is the naming maybe not precise.

Reply:
let's keep it. my suggestion is: `content recovery callback triggered` and then also include (at least) the hash of the chunk which is being requested for repair

Reply:
done. thanks for the feedback.